ecosystem/wallet-apps/testnet-tokens.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "How to get TON testnet tokens"
+sidebarTitle: "Fund testnet wallet"
+---
+
+TON testnet tokens are free test coins used for development and testing on the TON testnet network. They have no real value and are used instead of spending real TON.
+
+## Get tokens from the testnet faucet bot
+
+The TON testnet faucet is a TON Telegram bot that dispenses free test TON tokens.
+
+To request testnet tokens:
+
+1. Open [@testgiver\_ton\_bot](https://t.me/testgiver_ton_bot) in Telegram
+1. Start the bot by sending the `/start` command
+1. Tap the **Get 2 TON in testnet** button (or send the `/get` command)
+1. Complete the captcha that appears
+1. Provide your wallet address when prompted
+1. The bot will send 2 testnet TON to your address
+
+<Note>You can request test TON once every hour.</Note>
+
+<img
+  src="/resources/images/fund-wallet/testgiver.png"
+  alt="Testgiver TON Bot interface"
+  height="300"
+  className="rounded-lg"
+/>
+
+To interact with the bot at any time use the `/menu` command to view options, or send `/get` to quickly request more tokens.
+
+## Verify the tokens
+
+After requesting tokens, wait a few seconds for the transaction to complete. Then, check your wallet balance to confirm you received the testnet TON. You can also verify the transaction on the [Testnet explorer](https://testnet.tonscan.org/).
```

Comment:
please provide full example, this article needs to be self-contained and it should not presume full familiarity with TON blockchain: if you mention any checks, then there should be a screenshot of how it's supposed to look like, both in your wallet and in the testnet explorer